core/modules/layout/css/layout.admin.css

```diff
@@ -317,6 +323,9 @@
 .layout-access-list .layout-condition .layout-operations {
   display: inline-block;
 }
+.layout-content-form main {
+  margin-bottom: 0; /* Override Seven's styling. */
```

Comment:
Coding standards say that only `/* LTR */` is allowed next to CSS properties. This should go above the ruleset (in which case, a blank line above it is also required).

Reply:
You're right about the LTR inline comments, thanks for pointing that out. But I disagree with putting the comment above the ruleset as it just applies to that one property.

> Place the comment on the line immediately above the **property** or ruleset it describes. The comment should be indented the same amount as the **property** or ruleset it describes.
**If** the comment is describing a ruleset, place a blank line before the comment.
(emphasis mine)

So I think it's ok to put where I did, and without a blank line before it. PR updated.